tests/plugins/test_instantiation.py

```diff
@@ -11,38 +11,77 @@
 
 
 def test_load_plugin_from_yaml(mocker):
-    class SamplePlugin(object):
-        def __init__(self, banana, other):
-            self.something = banana
-            self.other = other
+    class SampleModule(object):
```

Comment:
[nit] I know this is being mocked anyway, but a Module is not exactly a class.
I think that it would make more sense in the test if you do `SampleModule = mocker.MagicMock(SamplePlugin=SamplePlugin)`. 

This is because it doesn't lead people reading this code in the future to imagine that `module_obj` _should_ be a class